src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -1921,6 +1921,7 @@
   "com.minecolonies.coremod.gui.flag.base_layer": "Base",
   "com.minecolonies.coremod.gui.townhall.edit_flag": "Edit Colony Flag",
   "block.minecolonies.colony_banner": "Colony Flag Banner",
+  "com.mincolonies.coremod.item.colony_baner.tooltipempty": "Right click to copy pattern on the item",
```

Comment:
Typo

Reply:
I'm dyslexic and don't see the typo, can you tell me whats wrong ?

Reply:
baner-> banner

Reply:
And mincolonies should probably be minecolonies

Reply:
thx